samples/project-anfield/resources.json

```diff
@@ -0,0 +1,9 @@
+{
+    "jobs": [],
```

Comment:
As discussed today, how are you making sure that the user isn't getting into error if they are deleting policy / entity before the job ?

Reply:
Importing jobs is specific